<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="playWorkingSound_checkBox">
     <property name="text">
      <string>play sound on working time timeout</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="playRestingSound_checkBox">
     <property name="text">
      <string>play sound on resting time timeout</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="showWorkingTrayNotify_checkBox">
     <property name="text">
      <string>show tray notify on working time timeout</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="showRestingTrayNotify_checkBox">
     <property name="text">
      <string>show tray notify on resting time timeout</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="saveChangesOnExit_checkBox">
     <property name="text">
      <string>automatic save changes on exit</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>playWorkingSound_checkBox</tabstop>
  <tabstop>playRestingSound_checkBox</tabstop>
  <tabstop>showWorkingTrayNotify_checkBox</tabstop>
  <tabstop>showRestingTrayNotify_checkBox</tabstop>
  <tabstop>saveChangesOnExit_checkBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
